datacite-delete-draft-doi-handler/src/main/java/no/unit/nva/datacite/handlers/DeleteDraftDoiAppEnv.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.datacite.handlers;
+
+import nva.commons.utils.Environment;
+import nva.commons.utils.JacocoGenerated;
+
+@JacocoGenerated
+public final class DeleteDraftDoiAppEnv {
```

Comment:
All these methods  are used only in one handler? Why to have a separate class? each class we create is some extra time in the cold start